### DISCUSSION TITLE
Combobox component providing countries (iso-3166) with flag

### DISCUSSION THREAD

**Discussion by umesh00** [shadcn-ui/ui#2276] [Ideas]
Hi,

I had recently started using shadcn and really love it. Recently I had a requirement to provide ISO 3166 country codes with flag and had to extend Comobox using Command and Popover. I would like to contribute back if that would be of any interest?

Some screenshots:
![Screenshot 2024-01-03 at 13 49 54](https://github.com/shadcn-ui/ui/assets/1090895/cff33acd-4cb2-4466-888a-afd973cadc45)

![Screenshot 2024-01-03 at 13 50 18](https://github.com/shadcn-ui/ui/assets/1090895/148f99b5-58bf-47db-b023-3f0f83672492)

![Screenshot 2024-01-03 at 13 50 52](https://github.com/shadcn-ui/ui/assets/1090895/607bcaba-6fcc-4abe-97dd-3bd33bdc8e32)

![Screenshot 2024-01-03 at 13 56 32](https://github.com/shadcn-ui/ui/assets/1090895/cfe7d17b-ac42-4c89-a4bf-feb6270241d3)

this component has below dependencies:
i18n-iso-countries
flag-icons

Thanks,
Umesh

**Comment by devmoatassem** [comment]
Hey Can you please share code?

**Comment by 0x113** [comment]
Looks amazing! Would you mind sharing your code?

**Comment by masterchief-Dave** [comment]
@devmoatassem @0x113 
You can check this open-source code on github [here](https://github.com/Jayprecode/country-state-dropdown).

This is also a curated list of components built with Shadcn [here](https://github.com/birobirobiro/awesome-shadcn-ui).
